src/test/java/uk/gov/hmcts/reform/professionalapi/service/impl/PaymentAccountServiceImplTest.java

```diff
@@ -100,49 +100,50 @@ public void retrievePaymentAccountsByPbaEmailWhenConfigTrue() {
 
         verify(organisationMock, times(1)).setPaymentAccounts(any());
         verify(organisationMock, times(1)).setUsers(any());
-        verify(organisationMock, times(1)).setPaymentAccounts(any());
         verify(professionalUserRepositoryMock, times(1)).findByEmailAddress("some-email");
     }
 
+    @Test(expected = Exception.class)
+    public void testThrowsExceptionWhenEmailInvalid() {
+        when(sut.findPaymentAccountsByEmail("some-email")).thenReturn(organisation);
+    }
+
     @Test
-    public void retrievePaymentAccountsByPbaEmailWhenConfigFalse() {
+    public void testEditPaymentAccountsByOrganisationIdentifier() {
+
         Organisation organisationMock = mock(Organisation.class);
         final List<PaymentAccount> paymentAccounts = new ArrayList<>();
         paymentAccounts.add(new PaymentAccount());
         ProfessionalUser professionalUserMock = mock(ProfessionalUser.class);
 
         when(professionalUserMock.getOrganisation()).thenReturn(organisationMock);
-        when(applicationConfigurationMock.getPbaFromUserAccountMap()).thenReturn("false");
         when(organisationMock.getStatus()).thenReturn(OrganisationStatus.ACTIVE);
         when(organisationMock.getPaymentAccounts()).thenReturn(paymentAccounts);
-        when(professionalUserRepositoryMock.findByEmailAddress("some-email")).thenReturn(professionalUserMock);
+        when(organisationRepositoryMock.findByOrganisationIdentifier(any(String.class))).thenReturn(organisation);
```

Comment:
can you add a `verify` for `organisationRepositoryMock.findByOrganisationIdentifier`

Reply:
ok